src/lib/types/events.ts

```diff
@@ -2094,6 +2097,20 @@ export class ReleasePlanTemplateDeletedEvent extends BaseEvent {
     }
 }
 
+export class ReleasePlanTemplateArchivedEvent extends BaseEvent {
+    readonly preData: any;
+    readonly data: any;
+    constructor(eventData: {
+        data: any;
+        preData: any;
+        auditUser: IAuditUser;
+    }) {
+        super(RELEASE_PLAN_TEMPLATE_ARCHIVED, eventData.auditUser);
+        this.data = eventData.data;
+        this.preData = eventData.preData;
```

Comment:
Should it have both data and preData? What would you expect to see in these fields?

See removed event below, for example.

Reply:
There will be an archived_at column in the data, so figured that warranted a data field